lib/Echidna/Config.hs

```diff
@@ -33,7 +33,7 @@ import qualified Data.Yaml as Y
 import Echidna.Solidity
 import Echidna.Test
 import Echidna.Types.Campaign (CampaignConf(CampaignConf))
-import Echidna.Types.Tx (TxConf(TxConf))
+import Echidna.Types.Tx
```

Comment:
let's continue using qualified imports

we can replace things with `(..)` if necessary

Reply:
Fixed.